internal/app/machined/pkg/system/system.go

```diff
@@ -18,7 +18,8 @@ import (
 	"github.com/talos-systems/talos/pkg/conditions"
 )
 
-type singleton struct {
```

Comment:
Which linter enforces this change? Keeping this type private was supposed to be the way to enforce single instance of this type

Reply:
let me check

Reply:
that's golint. I guess it makes sense to set `nolint:golint` there instead.